geometry/include/pcl/geometry/mesh_indices.h

```diff
@@ -425,156 +160,51 @@ class EdgeIndex
  */
-class FaceIndex
-: boost::totally_ordered<
-      pcl::geometry::FaceIndex // < > <= >= == !=
-      ,
-      boost::unit_steppable<pcl::geometry::FaceIndex // ++ -- (pre and post)
-                            ,
-                            boost::additive<pcl::geometry::FaceIndex // += + -= -
-                                            >>> {
-public:
-  using Base = boost::totally_ordered<
-      pcl::geometry::FaceIndex,
-      boost::unit_steppable<pcl::geometry::FaceIndex,
-                            boost::additive<pcl::geometry::FaceIndex>>>;
-  using Self = pcl::geometry::FaceIndex;
-
-  /** \brief Constructor. Initializes with an invalid index. */
-  FaceIndex() : index_(-1) {}
-
-  /** \brief Constructor.
-   * \param[in] index The integer index.
-   */
-  explicit FaceIndex(const int index) : index_(index) {}
-
-  /** \brief Returns true if the index is valid. */
-  inline bool
-  isValid() const
-  {
-    return (index_ >= 0);
-  }
-
-  /** \brief Invalidate the index. */
-  inline void
-  invalidate()
-  {
-    index_ = -1;
-  }
-
-  /** \brief Get the index. */
-  inline int
-  get() const
-  {
-    return (index_);
-  }
-
-  /** \brief Set the index. */
-  inline void
-  set(const int index)
-  {
-    index_ = index;
-  }
-
-  /** \brief Comparison operators (with boost::operators): < > <= >= */
-  inline bool
-  operator<(const Self& other) const
-  {
-    return (this->get() < other.get());
-  }
-
-  /** \brief Comparison operators (with boost::operators): == != */
-  inline bool
-  operator==(const Self& other) const
-  {
-    return (this->get() == other.get());
-  }
-
-  /** \brief Increment operators (with boost::operators): ++ (pre and post) */
-  inline Self&
-  operator++()
-  {
-    ++index_;
-    return (*this);
-  }
-
-  /** \brief Decrement operators (with boost::operators): \-\- (pre and post) */
-  inline Self&
-  operator--()
-  {
-    --index_;
-    return (*this);
-  }
-
-  /** \brief Addition operators (with boost::operators): + += */
-  inline Self&
-  operator+=(const Self& other)
-  {
-    index_ += other.get();
-    return (*this);
-  }
-
-  /** \brief Subtraction operators (with boost::operators): - -= */
-  inline Self&
-  operator-=(const Self& other)
-  {
-    index_ -= other.get();
-    return (*this);
-  }
-
-private:
-  /** \brief Stored index. */
-  int index_;
-
-  friend std::istream&
-  operator>>(std::istream& is, pcl::geometry::FaceIndex& index);
-};
-
-/** \brief ostream operator. */
-inline std::ostream&
-operator<<(std::ostream& os, const pcl::geometry::FaceIndex& index)
-{
-  return (os << index.get());
-}
-
-/** \brief istream operator. */
-inline std::istream&
-operator>>(std::istream& is, pcl::geometry::FaceIndex& index)
-{
-  return (is >> index.index_);
-}
+using VertexIndex = pcl::detail::MeshIndex<struct VertexIndexTag>;
+/** \brief Index used to access elements in the half-edge mesh. It is basically
+ * just a wrapper around an integer with a few added methods. \author Martin
```

Comment:
If you notice, the original comment has \author and \ingroup on new lines (which is required for doxygen). Could you please make the necessary changes?

Reply:
You are right. 
I've checked and it is modified when the clang format is applied on the file. Hope now it is ok.

Reply:
The issue is with windows CI not your code